tasks/upgrade_debian.yml

```diff
@@ -21,6 +21,11 @@
   debug: msg="{{ updates.stdout_lines | count }} packages to upgrade ({{ updates.stdout_lines | join(', ') }})"
   when: updates.stdout_lines | count > 1
 
+- name: Remove needrestart DPKG hook
```

Comment:
Please move this section to line 13, direct after the 'Install dependencies' task.

Reply:
Why? I placed it specifically directly before executing the `apt` task.

Reply:
I think is more appropriate to configure the system first(such thinks like installing all dependencies, write correct config files...) and do the regular stuff afterwards. Even more I thinks it's more clear to have all upgrade tast in a sequence, without any configuration in there.

Reply:
Fine.